ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/StatementRewriter.java

```diff
@@ -422,15 +415,13 @@ protected Node visitCreateTable(final CreateTable node, final Object context) {
   }
 
   protected Node visitCreateTableAsSelect(final CreateTableAsSelect node, final Object context) {
-    return new CreateTableAsSelect(node.getLocation(),
+    return new CreateTableAsSelect(
+        node.getLocation(),
         node.getName(),
         (Query) process(node.getQuery(), context),
         node.isNotExists(),
-        node.getProperties().entrySet().stream()
-            .collect(Collectors.toMap(
-                Entry::getKey,
-                e -> (Literal) process(e.getValue(), context)
-            )));
+        node.getProperties()
```

Comment:
As above, this does remove 'rewriting' of the properties. However, we don't use this anyway and the same is already true for C* properties.